From 48893e426ad7bddfd7b75b664854d4956bf5f76c Mon Sep 17 00:00:00 2001
From: Junwei Lu <Junwei Lu>
Date: Tue, 19 Jul 2016 22:57:03 +0800
Subject: [PATCH] mtt

---
 src/m.txt | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 src/m.txt

diff --git a/src/m.txt b/src/m.txt
new file mode 100644
index 0000000..37414a5
--- /dev/null
+++ b/src/m.txt
@@ -0,0 +1 @@
+m.tt
\ No newline at end of file
-- 
2.9.0.windows.1

